external/anomaly/anomaly_classification/configs/stfpm/configuration.yaml

```diff
@@ -113,7 +124,6 @@ pot_parameters:
       operator: AND
       rules: []
       type: UI_RULES
-    value: 300
```

Comment:
Why is this removed?

Reply:
I used [this script](https://gitlab.devtools.intel.com/vmc-eip/IMPT/impt/-/blob/develop/src/sc_sdk/ote_hyperparam_configuration_generator) from SC_SDK to generate the `configuration.yaml` files. Apparently some changes were made in how the yaml files get populated in this conversion process. I don't think it should be a problem, because the `default_value` field is still present.

Reply:
Alright thanks!